lib/jsdom/living/helpers/form-controls.js

```diff
@@ -58,6 +60,21 @@ exports.normalizeToCRLF = string => {
     .replace(/^\n/, "\r\n");
 };
 
+// https://html.spec.whatwg.org/multipage/forms.html#valid-e-mail-address
+exports.isValidEmailAddress = (emailAddress, multiple = false) => {
+  const emailAddressRegexp = new RegExp("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9]" +
+    "(?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}" +
+    "[a-zA-Z0-9])?)*$", "i");
```

Comment:
The `ignore case`-flag is not in the spec (and seems like it might not make a difference based on how the regex is written).